### ISSUE TITLE
PLSQL identifiers can be keywords

### ISSUE THREAD

**Issue by stephendrew** [antlr/grammars-v4#720]
Suggest making a rule for keyword tokens, and allow them to be used as identifiers.

**Comment by stephendrew** [comment]
I believe this will fix #553 and #643

**Comment by stephendrew** [comment]
Actually, I notice some attempt has been made at adding these keywords to the `regular_id` already, but the list seems incorrect (e.g. `DELETE` is a keyword and can be an identifier).

**Comment by KvanTTT** [comment]
You are right: `regular_id` contains some keywords. It would be great if you add missing and remove wrong keywords from this rule :)

**Comment by stephendrew** [comment]
In my divergent grammar, which I am trying to reconcile with this, I had a separate `plsql_keyword` (`plsql_reserved_word`) rule - which approach do you prefer - using such a rule?  Or simply adding to `regular_id`...?

**Comment by KvanTTT** [comment]
I like your approach.
